src/util-inl.h

```diff
@@ -27,6 +27,7 @@
 #include <cmath>
 #include <cstring>
 #include "util.h"
+#include "base64-inl.h"
```

Comment:
It’s not common practice to include another header just so that its definitions are also available from here – it’s probably better to remove this line and add it to the source files that use the `ReadUint32BE` function?